apps/api/src/form/entity/form.entity.ts

```diff
@@ -6,7 +6,7 @@ import { FormPayloadDTO } from '@ehpr/common';
 @Entity('form')
 export class FormEntity extends BaseEntity {
   @Exclude()
-  @Column('jsonb', { nullable: false })
+  @Column('jsonb', { nullable: true })
```

Comment:
@arranfw @hitankar-freshworks @vyasworks @dbayly-freshworks 
This is not required we can revert this.